### PR TITLE
Fixed being unable to login to security console

### LINKED ISSUE #8387: Security console don't open and keep id
### Environment
- Game Version: UnityStationDevelop 2 2021 801
- OS Version: Lubuntu 18.04.15x64

### Detailed description of bug

the user interface of the security department is broken, 
it can be opened and closed.
but once an ID is inserted in it, the login button does not work. 
And no way to get your card back.
You have to replace it with another card.

### Steps to reproduce

1. insert your ID card on sec console
2. can't login
3. stay blocked like this

### PR DESCRIPTION
**Pull request by MaxIsJoe**
Fixes #8387

CL: [Fix] Fixed being unable to login to security console

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Objects/Security/SecurityRecordsConsole/GUI_SecurityRecords.cs [hunk 1]
```diff
@@ -87,7 +87,7 @@ public void UpdateIdText(NetLabel labelToSet)
 
 		public void LogIn()
 		{
-			if (console.IdCard == null || !console.IdCard.HasAccess(Access.security) || IsAIInteracting() == false)
+			if ((console.IdCard == null && IsAIInteracting() == false) || !console.IdCard.HasAccess(Access.security))
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/UI/Objects/Security/SecurityRecordsConsole/GUI_SecurityRecords.cs]: truth table

Perhaps you want this?
```suggestion
			if ((console.IdCard == null || console.IdCard.HasAccess(Access.security) == false) && IsAIInteracting() == false)
```

**Comment by MaxIsJoe** [comment]
Tested Nooney's change. it works for both human characters and the AI.
approvals pls.

**Comment by Bod9001** [comment]
update branch for tests

**Comment by MaxIsJoe** [comment]
Rebased

**Review by NoooneyDude — APPROVED** [review]

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**